state/cleanup.go

```diff
@@ -541,6 +541,14 @@ func cleanupDyingMachineResources(m *Machine) error {
 		return errors.Annotate(err, "getting machine volume attachments")
 	}
 	for _, va := range volumeAttachments {
+		detachable, err := isDetachableVolumeTag(m.st, va.Volume())
```

Comment:
if `detachable` isn't needed beyond here then I'd tend to roll the call into the `if` so that `detachable` doesn't escape, and you save a few lines.